index.html

```diff
@@ -68,7 +69,204 @@ <h2> I’m a software developer</h2>
       <div class="chevron-btn">
         <img class="all-image" src="images/Action.svg" alt="arrow down image">
       </div>
+
+      <div class="text">
+        <h4>My Recent<br> Works</h4>
+      </div>
+
+      <div class="an-image">
+        <img src="images/Indicator.svg" alt="">
+      </div>
+    </section>
+
+    <section class="Portfolio">
```

Comment:
- [ ] Please let's consider adding some more margin-left and right on the `my works` section because for now, the projects are so close to the left and right sides, 👍